examples/ray_orchestrator/rl_perf_repro.py

```
@@ -0,0 +1,274 @@
+            """Generate a single prompt asynchronously"""
+            object_ref = instance.generate.remote(prompt=prompt)
+            result = await asyncio.to_thread(ray.get, object_ref)
+            return result
+
+        # Create tasks with round-robin distribution
+        tasks = [
+            generate_single_prompt(llm_instances[idx % num_instances], prompt)
+            for idx, prompt in enumerate(prompts)
+        ]
+
+        results = await asyncio.gather(*tasks)
```

Ruff (0.14.8)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)